Pod/Classes/APMAlertController.swift

```diff
@@ -25,10 +26,10 @@ import SnapKit
 
     let alertView = UIView()
     fileprivate let topScrollView = UIScrollView()
-    fileprivate var topScrollViewHeightConstraint: Constraint?
+    fileprivate var topScrollViewHeightConstraint: NSLayoutConstraint?
     fileprivate let contentView = UIView()
     fileprivate var anyTitleObject: AnyObject
-    fileprivate var topTitleMessageSeparatorConstraint: Constraint?
+    fileprivate var titleMessageSeparatorConstraint: NSLayoutConstraint?
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/almas-dev/APMAlertController/blob/7b25c9b676cbcd14f599e9b699c32cd1617e4f78/Pod/Classes/APMAlertController.swift#L32"></span>Variable name should be between 2 and 24 characters long: ‘titleMessageSeparatorConstraint’</td>
    </tr>
  </tbody>
</table>